apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts

```diff
@@ -899,6 +903,10 @@ export const functions: NavMenuConstant = {
       url: undefined,
       items: [
         { name: 'Dart Edge on Supabase', url: '/guides/functions/dart-edge' },
+        {
+          name: 'Expo Push Notifications',
+          url: '/guides/functions/examples/push-notifications',
+        },
```

Comment:
Would it be better to just have a single menu item for push notifications? We could probably leave the one at the top?